manifests/03-rbac-role-cluster.yaml

```diff
@@ -105,6 +105,12 @@ rules:
   - validatingwebhookconfigurations
   verbs:
   - get
+- apiGroups:
+  - operators.coreos.com
+  resources:
+  - subscriptions
+  verbs:
+  - get
```

Comment:
It's better to restrict the permission to the single subscription name / namespace we need if it's a well-known value.

Reply:
yes, please add `resourceNames` field with appropriate name of the subscription instance.

Reply:
I've added in the resourceNames field